src/schema/sources.ts

```diff
@@ -1099,16 +1099,16 @@ export const resolvers: IResolvers<any, Context> = {
       };
 
       const subQuery = await ctx.con.query(
-        `with base as (SELECT * FROM source_tag_view), s as (
+        `with s as (
             SELECT *, row_number() over (partition by "sourceId" order by count desc) rn
-            FROM base
+            FROM source_tag_view
         )
-        SELECT s2."sourceId", s2."count"
+        SELECT s2."sourceId"
         FROM s s1
         JOIN s s2 on s1.tag = s2.tag and s1."sourceId" != s2."sourceId"
         WHERE s1."sourceId" = $1 and s1.rn <= 10 and s2.rn <= 10
-        GROUP BY 1, s2."count"
-        ORDER BY s2."count" desc
+        GROUP BY 1
+        ORDER BY count(*) desc
```

Comment:
Ante asked me to use count here, this is why I added it.
Don't think it makes the query different right?

Reply:
It's not the same value. This is why actually the production query prefers sources with lots of posts.
Basically I'm checking here how many of the top 10 tags are similar to the source in the request.
Using `s2.count` changes completely the logic